### ISSUE TITLE
questions: servie mesh in clusternet

### ISSUE THREAD

**Issue by hangz1995** [clusternet/clusternet#547]
Is there any plan to support servie mesh (such as Istio) in clusternet HUB?

**Comment by dixudx** [comment]
@hangz1995 Yes, it is on our plan. Hopefully that will be available in the coming two releases.

Any interests to help integrate this feature?

**Comment by hangz1995** [comment]
> 



> @hangz1995 Yes, it is on our plan. Hopefully that will be available in the coming two releases.
> 
> Any interests to help integrate this feature?

I will try to do some research, thank you~
